cmd/release/main.go

```diff
@@ -288,7 +320,7 @@ func (g git) tag(version string) error {
 		// assume this means it doesn't exist
 		cmd = exec.Command("git", "tag", "-s", "-m", fmt.Sprintf("A release of %q for version %q", g.repository, version), version)
 		out, err2 := cmd.CombinedOutput()
-		if err != nil {
+		if err2 != nil {
 			fmt.Println(string(out))
```

Comment:
any reason not to use:
```go
if out, err := cmd.CombinedOutput(); err != nil {
  fmt.Println(string(out))
  return err
}
```

Reply:
No, it's just a style thing. My reason for writing it this way is that I'd be returning `err` regardless if it's a nil or not but I only want to print when the `err2` is not nil